tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "lib": [
       "dom",
       "dom.iterable",
-      "esnext"
+      "esnext",
+      "ES2023.Array",
```

Comment:
This `tsconfig` is incompatible with the TS version used by this project (4.9). It might work if you're telling VS Code to use its currently shipped TS version but I'd consider it good practice to ensure the project typechecks and builds with the TS version actually declared in dependencies.

Updating TS is a bit of a pain since (see above) CRA is unmaintained...